### PR TITLE
ci: Add workflow to trigger Athena build

### PR DESCRIPTION
**Pull request by paulgessinger**
Only runs on the main branch, uses a secret trigger token to build it

### CODE REVIEW

File: .github/workflows/trigger_athena.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+name: Trigger Athena build
```
Comment (CarloVarni) [inline .github/workflows/trigger_athena.yml:1]: I'd suggest not to use `trigger` in the name of the test and file. That may suggest this is related to the Athena trigger.

**Review by CarloVarni — COMMENTED** [review]
To understand... this will add a test to the CI that checks the Athena build is not broken to the PR?

File: .github/workflows/trigger_athena.yml (hunk 1 above)
Comment (paulgessinger) [reply .github/workflows/trigger_athena.yml:1]: I see. It's literally called a "pipeline trigger" in GitLab 😅

**Comment by paulgessinger** [comment]
> To understand... this will add a test to the CI that checks the Athena build is not broken to the PR?

Unfortunately no. This only makes sure that one CI job on GitLab runs for every commit to the main branch here.

File: .github/workflows/trigger_athena.yml (hunk 1 above)
Comment (paulgessinger) [reply .github/workflows/trigger_athena.yml:1]: Can we keep this? I think it's standard terminology.

**Review by CarloVarni — APPROVED** [review]

**Pull request merged**